CHANGES.txt

```diff
@@ -16,17 +16,32 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Whatever John Doe did.
 
   From Joseph Brill:
-    - MSVS: Fix a significant MSVC/MSVC tool initialization slowdown when
-      vcpkg has been installed and the PSModulePath is not initialized
-      or propagated from the user's shell environment. To resolve this
-      The default Windows Powershell 7 path is added before the default
-      Windows Powershell 5 path in the carefully constructed
-      environment in which the MSVC batch files are run.   The shell
-      environment variables and values for VCPKG_DISABLE_METRICS and
-      VCPKG_ROOT are added to the limited MSVC environment when defined.
-      At present, the VCPKG_DISABLE_METRICS and VCPKG_ROOT variables and
-      values are not propagated to the SCons environment after running
-      the MSVC batch files.
+    - MSVC: A significant delay was experienced in the windows 2022
+      and 2025 runners due to the limited environment in which the MSVC
```

Comment:
can you add "GitHub Actions runners', instead of runners?

Reply:
Reworked all text.